### ISSUE TITLE
Performance doesn't match Bolt.new

### ISSUE THREAD

**Issue by Reiss-Cashmore** [coleam00/bolt.new-any-llm#286]
### Describe the bug

Hi, 

If you compare performance with just bolt.new responses with the bolt.new running locally using Cloud Claude 3.5 Sonnet the responses are so much worse and amount of time it trips up in the IDE is so much higher in the local running version. 

Is the local version gimped in some way compared to the cloud version and is this expected or are we missing something when it comes to the setup of the local version? Perhaps the initial context passed to the model is different?

Does anyone else notice this? The cloud performance is just so much better, less errors, less trip ups, failure to run commands.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-2tmosp

### Steps to reproduce

Initial Prompt: Make me a simple one page hello world mobile app using react nativescript that I can run and preview using the nativescript preview app. Make sure its configured and set up with tailwind

Second Prompt: Make an very professional looking dashboard for a smart home control application that can remotely control all of the various systems you would find in a campervan conversion

Compare responses between local and cloud. Local significantly worse

### Expected behavior

That the local version matches the cloud version when using Claude 3.5 Sonnet directly from Anthropic

### Platform

- OS: Mac OS
- Browser: Chrome
- Version: Chrome 130

**Comment by rottingcorpse** [comment]
Cloud version has lot more processing power serving it. Your local models will always be much slower in terms of tokens/sec, unless you want to spend some millions of USD to buy some Nvidia server AI/GPU hardware.

TLDR; your hardware is much less powerful than one, that serves cloud versions of AI models

**Comment by Donubu** [comment]
i think Bolt.new use a different version or implementation of webcontainer. In this repository, we only have access to the free plan.

**Comment by frogman544** [comment]
What happens when we reach the API limit for the free tier? 
Is there a plan here to make this sustainable?

**Comment by Reiss-Cashmore** [comment]
> Cloud version has lot more processing power serving it. Your local models will always be much slower in terms of tokens/sec, unless you want to spend some millions of USD to buy some Nvidia server AI/GPU hardware.
> 
> TLDR; your hardware is much less powerful than one, that serves cloud versions of AI models

This is the thing though, I'm talking about the Local Bolt.new UI running using the Anthropic Hosted API Cloud Claude 3.5 Sonnet. Not a _self-hosted_ AI model. Sonnet is the same one that is being used by the _Cloud_ Bolt.new I believe. So why is the performance so much worse when the only variable that is changed is running bolt.new locally vs using bolt.new in the cloud
